src/app/tasks/task-submission-viewer/task-submission-viewer.component.ts

```diff
@@ -0,0 +1,41 @@
+import { Component, OnInit, Input, SimpleChanges, Inject} from '@angular/core';
+import { TaskFeedback } from 'src/app/ajs-upgraded-providers';
+
+@Component({
+  selector: 'app-task-submission-viewer',
+  templateUrl: './task-submission-viewer.component.html',
+  styleUrls: ['./task-submission-viewer.component.scss']
+})
+
+export class TaskSubmissionViewerComponent implements OnInit {
+
+  @Input() project: any;
+  @Input() task: any;
+
+  taskUrl: string;
+  taskFilesUrl: string;
+  
+  constructor(
+    @Inject(TaskFeedback) private TaskFeedback: any
+  ) { }
+
+  ngOnInit(): void {
+  }
+
+  notSubmitted(task): boolean {
+    return (!task.has_pdf && (!task.processing_pdf))
+  }
+
+  loadingDetails(task): boolean {
+    return (task.needsSubmissionDetails())
```

Comment:
Do we need this method? Can we just use the method on task in the directive?

Reply:
The above has been removed